extra/man/alacritty.5.scd

```diff
@@ -61,6 +61,7 @@ override variables set by alacritty itself.
 ```
 [env]
 TERM = "alacritty"
+PREFER_EGL = "1"
```

Comment:
Maybe we should document this in the `debug` section as well... Or somehow en `env` section, what do you think @chrisduerr ? Maybe we should document other alacritty specific variables?

Reply:
There should not be an environment variable for it. Just a debug flag, that's it.

Environment variables are just hacky.

Reply:
Yeah, I just thought about env variable, since initially I wanted it inside the `glutin`, but in alacritty it's indeed doesn't make that much sense.